tests/ConfigurationTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Doctrine\AnnotationsTests;
+
+use Doctrine\Annotations\Configuration;
+
+class ConfigurationTest extends TestCase
+{
+    public function testConfigurationAccessors()
+    {
+        $metadata  = $this->getMock('Doctrine\Annotations\Metadata\MetadataFactory', [], [], '', false);
```

Comment:
I suggest doing this change in the whole testsuite, but in a separate PR. This will allow to review the v2 rewriting of the library without having all these changes in the diff, making it easier to spot what actually changed in the testsuite due to the rewriting (allowing to spot BC breaks).